static/css/main.css

```diff
@@ -588,3 +588,17 @@ table td {
   height: 1px;
   overflow: hidden;
 }
+
+.prev-post {
+    float: left;
+    text-align: left;
+}
+
+.next-post {
+    float: right;
+    text-align: right;
+}
+
+.prev-post, .next-post {
+    max-width: 33%;
```

Comment:
nit: please fix indent level to 2 spaces

Reply:
Fixed by 7f70241f0e94171946826d39894a9f1995d01aea.